actioncable/lib/action_cable/channel/streams.rb

```diff
@@ -84,7 +84,8 @@ def stream_from(broadcasting, callback = nil, coder: nil, &block)
 
         connection.server.event_loop.post do
           pubsub.subscribe(broadcasting, handler, lambda do
-            transmit_subscription_confirmation
+            @defer_subscription_confirmation_counter.decrement
+            transmit_subscription_confirmation unless defer_subscription_confirmation?
```

Comment:
Let's name this, and push it into `base.rb` -- I know this is just a concern on the same class, but it still feels like we're getting too deep into details that don't belong here